examples/encoding/plot_continuous_encoder.py

```diff
@@ -0,0 +1,115 @@
+"""
+=============================
+Regression on continuous data
+=============================
+
+This demonstrates how encoding models can be fit with multiple continuous
+inputs. In this case, a encoding model is fit for one electrode, using the
+values of all other electrodes as inputs. These inputs could also be stimulus
+features, such as spectral features of sound.
+
+Estimation of receptive fields and modeling with continuous inputs
+is described in:
+
+    Theunissen, F. E. et al. Estimating spatio-temporal receptive
+    fields of auditory and visual neurons from their responses to
+    natural stimuli. Network 12, 289-316 (2001).
+
+    Willmore, B. & Smyth, D. Methods for first-order kernel
+    estimation: simple-cell receptive fields from responses to
+    natural scenes. Network 14, 553-77 (2003).
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+
+import mne
+import numpy as np
+from mne.datasets import sample
+from mne.encoding import SampleMasker, get_coefs
+from mne.channels.layout import _auto_topomap_coords
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import Ridge
+from sklearn.preprocessing import StandardScaler, scale
+
+# Load and preprocess data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+raw = mne.io.read_raw_fif(raw_fname, preload=True).pick_types(
+    meg=False, stim=False, eeg=True).filter(1, None, method='iir')
+ix_y = 10
+ixs_X = np.setdiff1d(range(len(raw.ch_names)), [ix_y])
+
+# Set up encoding model variables
+X = raw[:][0][ixs_X]
+y = raw[:][0][ix_y]
+
+f, axs = plt.subplots(2, 1, figsize=(10, 10), sharex=True)
+ix_plt = 200
+axs[0].plot(raw.times[:ix_plt], X[:, :ix_plt].T, color='k', alpha=.2)
+axs[0].set_title('Continuous Input Data')
+axs[0].set_xlim([0, raw.times[:ix_plt][-1]])
+axs[1].plot(raw.times[:ix_plt], y[:ix_plt], color='k')
+axs[1].set_title('Continuous Output Data')
+axs[1].set_xlabel('Time (s)')
+axs[1].set_ylabel('Amplitude (mV)')
+
+
+# --- Fit the model / Make predictions ---
+# Define model preprocessing and parameters
+ix_split = int(X.shape[-1] * .998)  # Using 99.8% of the data for pred viz
+tr = np.arange(X.shape[-1])[:ix_split]
+tt = np.arange(X.shape[-1])[ix_split:]
+
+# This is the final estimator that will be called after masking data
+pipe_est = make_pipeline(StandardScaler(), Ridge(alpha=0.))
```

Comment:
No - the results are the same, but the algorithm is different.

LinearRegression uses `linalg.lstsq` (for dense data, otherwise it uses a sparse lstsq), which means it calculates residuals, rank, and is generally much less efficient, IIRC particularly with multiple targets.